task/git-clone/0.6/git-clone.yaml

```diff
@@ -180,8 +186,12 @@ spec:
           chmod -R 400 "${PARAM_USER_HOME}"/.ssh/*
         fi
 
+        echo "what is the file location"
+        ls -lrt ${WORKSPACE_SSL_CA_DIRECTORY_PATH}
```

Comment:
do we need this?

Reply:
Yes added to check the file list in the current directory

Removed